Sources/CvGlobals.h

```diff
@@ -682,61 +682,61 @@ class cvInternalGlobals
 /* Afforess                                END                                                  */
 /************************************************************************************************/
 
-	int getNumCityTabInfos();
+	int getNumCityTabInfos() const;
 	std::vector<CvInfoBase*>& getCityTabInfos();
-	CvInfoBase& getCityTabInfo(CityTabTypes e);
+	CvInfoBase& getCityTabInfo(CityTabTypes e) const;
```

Comment:
I think all these functions should return const references, doubly so if the object itself is const. We shouldn't ever be modifying the CvInfo classes by definition right? We do it during loading I think, so we need some way to do it though...

Reply:
Step one is to add as much const to CvInfos as possible. (other pr has a lot)
I'll continue this as soon as I can.